phpunit.xml.dist

```diff
@@ -8,4 +8,14 @@
     <logging>
         <log type="coverage-text" target="php://stdout"/>
     </logging>
-</phpunit>
\ No newline at end of file
+
+    <filter>
+        <whitelist>
+            <directory>src</directory>
+            <exclude>
+                <directory>src/Google/Ads/GoogleAds/V*</directory>
+                <directory>src/Google/Ads/GoogleAds/Util/Testing</directory>
```

Comment:
Should we also exclude "src/Google/Ads/GoogleAds/Lib/Testing"?

Reply:
Done.